examples_tests/test_ros.py

```diff
@@ -0,0 +1,36 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2015, 2016 Canonical Ltd
```

Comment:
This is code from 2015, just moved to a separate file now. So I think it should have the two years.